CODE/logicytics/Checks.py

```
@@ -0,0 +1,68 @@
+
+            if zip_file and not ignore_file:
+                with zipfile.ZipFile(
+                        "../SysInternal_Suite/SysInternal_Suite.zip"
+                ) as zip_ref:
+                    zip_ref.extractall("SysInternal_Suite")
+                    return "SysInternal_Suite zip extracted"
+
+            elif ignore_file:
+                return "Found .sys.ignore file, skipping SysInternal_Suite zip extraction"
+
+        except Exception as err:
```

Ruff (0.8.0)

Do not catch blind exception: `Exception`

(BLE001)